docs/reference/modules/http.md

```diff
@@ -0,0 +1,186 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/http`
+
+**Alias:** `apos.http`
+
+This module adds methods for easy HTTP requests in Apostrophe project and module server-side code. It also defines error codes for use with the main `apos.error()` method.
+<!-- TODO: Link to the error module reference. -->
+
+**Extends:** `{{ $frontmatter.extends }}`
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+|`addErrors` | Object | An object of error names and their associated HTTP response codes. |
+
+### `addErrors`
+
+<!-- TODO: Link to the error module reference. -->
+This module's named errors are used in the `apos.error()` method. The default error names and their codes in Apostrophe are:
+- `min`: 400
+- `max`: 400
+- `invalid`: 400
+- `forbidden`: 403
+- `notfound`: 404
+- `required`: 422
+- `conflict`: 409
+- `locked`: 409
+- `unprocessable`: 422
+- `unimplemented`: 501
+
+Unnamed errors are returned with the 500 error code, so it does not need to be registered. Additional error codes can be added with the `addErrors` option.
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    options: {
+      addErrors: {
+        timeout: 408
+      }
+    }
+  }
+  ```
+  <template v-slot:caption>
+    modules/@apostrophecms/http/index.js
+  </template>
+</AposCodeBlock>
+
+## Related documentation
+
+- [Front end HTTP helper methods](/guide/front-end-helpers.md#http-request-methods)
+
+## Featured methods
+
+The following methods belong to this module and may be useful in project-level code. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/http/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+Because this module has an alias, you can call these from another module from the alias path. For example, `self.apos.http.addError()`.
+
+### `addError(name, code)`
+
+An alternate way to register error names to HTTP error codes. Pass the semantic name as the first argument, followed by the `code` integer.
```

Comment:
... which becomes the HTTP status code if that error is thrown in an Apostrophe route via `throw self.apos.error('errorname')`.